js/app/modules/card/knowledgeDocument.js

```diff
@@ -69,18 +69,10 @@ const KnowledgeDocument = new Module.Class({
             GObject.ParamFlags.READABLE,
             TableOfContents.TableOfContents.$gtype),
         /**
-         * Property: previous-card
```

Comment:
The commit message for this commit should explain why previous card is being removed and that it's not being replaced.